gtsam/hybrid/tests/testHybridGaussianISAM.cpp

```diff
@@ -227,7 +180,7 @@ TEST(HybridGaussianElimination, Approx_inference) {
   }
 
   // Now we calculate the actual factors using full elimination
-  const auto [unprunedHybridBayesTree, unprunedRemainingGraph] =
+  const auto [unPrunedHybridBayesTree, unPrunedRemainingGraph] =
```

Comment:
Nitpick: Isn't unpruned one word?

Source: https://www.oed.com/dictionary/unpruned_adj1?tl=true

Reply:
eh. My code spellchecker complained. Will leave as is as otherwise another CI :-/

Reply:
I can update this in a follow up. :)